.github/workflows/pull-request-automation.yml

```diff
@@ -2,12 +2,15 @@ on:
     pull_request_target:
         types: [opened]
     push:
+    workflow_run:
+        workflows: [Build Gutenberg Plugin Zip]
+        types: [completed, in_progress]
 name: Pull request automation
 
 jobs:
     pull-request-automation:
         runs-on: ubuntu-latest
-        if: ${{ github.repository == 'WordPress/gutenberg' }}
+        if: ${{ true || github.repository == 'WordPress/gutenberg' }}
```

Comment:
Note: I made this always true to test on personal repo, should be removed on trunk